v2/internal/util/annotations/parse.go

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT license.
+ */
+
+package annotations
```

Comment:
Putting these into a new package is perfect, but does it need the intermediate `util` folder? Suggest moving these up one level.

Reply:
We do already have a bunch of packages in the `util` folder, but I've moved these out